content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -193,6 +193,84 @@ it does the following when a Pod is created:
 1. If the spec of the incoming Pod doesn't already contain any `imagePullSecrets`, then the
    admission controller adds `imagePullSecrets`, copying them from the `ServiceAccount`.
 
+### Legacy ServiceAccount token cleaner
+
+{{< feature-state for_k8s_version="v1.29" state="beta" >}}
+
+Starting from Kubernetes version 1.28, the feature `LegacyServiceAccountTokenTracking`
+is enabled by default, which creates a ConfigMap that records the configmap
+creation date as the tracking date. With the feature enabled, each use of the
```

Comment:
This is confusing. Why are we creating a ConfigMap in this context?
What's the the namespace, name of the ConfigMap?
We are simply creating a ConfigMap for recording a tracking date?

Reply:
Hi Qiming, thank you very much for your comments. I added some detailed information of the ConfigMap. I need to explain the ConfigMap here as it will be used by the cleaner to check the tracking date to determine whether it's time to start to evaluate the secrets. If I don't explain the ConfigMap, the user will be confused and wonder why the cleaner has not started to mark some secret as invalid.

Reply:
This is an awful design, I'd say. ConfigMaps are containers for things that are meant to be exposed to users. Having a ConfigMap in the system, without knowing that it is being used by a builtin controller, I'd call this a confusing design. I'd prefer to record the timestamps as annotations on those objects to be cleaned. I believe many people like me would clean ConfigMaps we don't recognize.
If the design cannot be changed, we have to warn users that it is a special ConfigMap, don't touch it, don't modify it or remove it. I don't think really care about the facilities used for tracking, be it a ConfigMap or something else. All they may want to know is that there is a controller to track this.

Reply:
the configmap is namespaced in `kube-system`. all stuff within `kube-system` is not supposed be modified/deleted if not knowing what's it is doing. e.g. do you want to delete `kube-system/kube-root-ca.crt`?

Reply:
Does this ConfigMap have a well-known name?
I can guess the why the `kube-root-ca.crt` ConfigMap is there, and I won't touch it unless necessary.

Reply:
Hi Qiming, it's name is `kube-apiserver-legacy-service-account-token-tracking`. I have updated this name in the doc:)

Reply:
sigh ...